### ISSUE TITLE
Table count is off in Functions can be linear

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#1944]
[here's the deck](https://docs.google.com/presentation/d/10aAvqMdWR3RXc078HANwIp5gD3uVL1a4FpZfn9Rfsq0/edit#slide=id.139f96dc-dbba-11ee-9d3d-2de735153b50)

This code
```
On a graph, the y-intercept is the value where the line "intercepts" the y-axis.

[cols="^1,^1"]
|===
|
@centered-image{images/y-intercept-diagonal.png}
|
@centered-image{images/y-intercept-horizontal.png}
|===
```

generates this on slide 21, which is definitely from somewhere else and doesn't make sense here.

<img width="921" alt="Screen Shot 2024-03-06 at 7 19 36 AM" src="https://github.com/bootstrapworld/curriculum/assets/66691020/eb510f87-d911-432c-acd6-82bcf2251cd4">

**Comment by ds26gte** [comment]
Fix for #1942 should catch this too

**Comment by schanzer** [comment]
Confirmed fixed on `functions-can-be-linear`
